### DISCUSSION TITLE
[Feature] Show path for external library images

### DISCUSSION THREAD

**Discussion by enboig** [immich-app/immich#7951] [Feature Request]
### The feature

I would like to know the path of external library images. I added my whole old image library to immich and I discovered there are a lot of images that I should remove or send to other people. Checking info I get some metadata but not he file path.

Thanks.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by jenda69** [comment]
There's (i) icon next to the file name in the Info sidebar. When you click it, it shows the path. It is the path of the file inside Docker, but there's no problem to extrapolate a real path from that.
